test/e2e/specs/cart.overview.spec.js

```diff
@@ -0,0 +1,74 @@
+import AppHeader from '../page-objects/AppHeader';
+import SwagOverview from '../page-objects/SwagOverview';
+import CartOverview from '../page-objects/CartOverview';
+import CheckoutPersonalInfo from '../page-objects/CheckoutPersonalInfo';
+import {prepareEnvironment} from '../helpers';
+
+describe('Cart', () => {
+    it('should validate that we can continue shopping', () => {
+        prepareEnvironment('/cart.html');
+
+        CartOverview.waitForIsDisplayed();
+
+        // Actual test starts here
+        expect(SwagOverview.isDisplayed()).toEqual(
+            false,
+            'Inventory screen is already visible'
+        );
```

Comment:
Why do you need to have this explicitly? We just waited to be on CartOverview, there shouldn't be any way for it to be on SwagOverview if the previous line didn't error out.